### ISSUE TITLE
Vectorize for loop

### ISSUE THREAD

**Issue by zgyr** [phernst/pytorch_radon#3]
Is it possible to unwrap [this](https://github.com/phernst/pytorch_radon/blob/master/pytorch_radon/radon.py#L39) transformation loop?
If it is possible to collect all the slices into one tensor and send for processing, it should be more efficient.
And isn't another there a faster way to do this transform?

**Comment by phernst** [comment]
Yes, that's possible, see [here](https://github.com/HazyResearch/pytorch_radon/blob/master/pytorch_radon/radon.py#L31-L34). The problem is that `self.all_grids` is precomputed and can become pretty large depending on the number of projections and their size. Pushing it onto the gpu as a whole wouldn't work then. Transfering single grids to the gpu allows larger/more grids but decreases the speed, of course.
